frontend/src/pages/sign-in/SignInPage.tsx

```diff
@@ -0,0 +1,3 @@
+export default function SignInPage() {
+    return <>sign-in page</>
```

Comment:
Missing HTML tags, is that normal?

Reply:
It compiles correctly because `<></>` tags are a shorthand for `React.fragment`. But I put this up just to test if the route was actually working